Assets/Scripts/Player/PlayerCheckpoint.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class PlayerCheckpoint : MonoBehaviour
+{
+    public static int balance = 0;
+    public static float health = 100;
+    public static HashSet<Item.ItemType> items;
+    public static Vector3 location;
+    public static Quaternion rotation;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    private void OnTriggerEnter(Collider other)
```

Comment:
This confuses me, checkpoints should be programmatically saved not by physical location but when the player places an order at the kiosk, so having it triggered by a physical location doesn't really make sense